core/src/connection/handler.rs

```diff
@@ -28,8 +28,8 @@ use std::{fmt::Debug, task::Context, task::Poll};
 pub trait ConnectionHandler {
     /// The inbound type of events used to notify the handler through the `Network`.
     ///
-    /// See also [`EstablishedConnection::notify_handler`](super::EstablishedConnection::notify_handler)
-    /// and [`ConnectionHandler::inject_event`].
+    /// See also [`EstablishedConnection::notify_handler`](super::EstablishedConnection::
+    /// notify_handler) and [`ConnectionHandler::inject_event`].
```

Comment:
This change seems to be breaking CI, more specifically it seems to not be a valid intra doc link.

Reply:
Yes, I guess there is a reason why this setting is unstable :(

If I manage to change these docs in a way so the formatting doesn't break the link: Would this be acceptable to merge?